lndr.admin.inc

```diff
@@ -9,6 +9,15 @@
  * Form builder callback.
  */
 function lndr_admin_settings($form, &$form_state) {
+  global $base_url;
+  $form['lndr_base_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Base URL'),
+    '#default_value' => variable_get('lndr_base_url', $base_url),
+    '#description' => t("Allows to override a base url setting."),
```

Comment:
There should be a better explanation of this configuration option to the end user.
1. Explain that in most cases this does not need to be changed at all
2. Why this needs to be changed, in what situation this should be changed
3. Possibly think about moving this option down so it's better UX when user configure the module